jsettlers.logic/src/jsettlers/logic/movable/Movable.java

```diff
@@ -708,6 +692,13 @@ private void setState(EMovableState newState) {
 		this.state = newState;
 	}
 
+	private void setStrategy(MovableStrategy newStrategy) {
```

Comment:
No, please don't reorder all the methods...
1. his makes it impossible to somehow review this PR
2. most of the internal methods are grouped together because they belong together and are used by each other. Therefore this makes reading, understanding and working in the code much easier as when simply grouping the methods. When reordering something, be aware that the order you see is mostly intended.